app/channels/notifications_channel.rb

```diff
@@ -0,0 +1,5 @@
+class NotificationsChannel < ApplicationCable::Channel
```

Comment:
That's handled in `connection.rb` [here](https://github.com/codeforamerica/clientcomm-rails/blob/master/app/channels/application_cable/connection.rb#L12-L19)